eng/jobs/prepare-signed-artifacts.yml

```diff
@@ -0,0 +1,60 @@
+parameters:
+  dependsOn: []
+  PublishRidAgnosticPackagesFromJobName: ''
+
+jobs:
+- job: PrepareSignedArtifacts
+  displayName: PrepareSignedArtifacts
```

Comment:
(nit) There is no real value in declaring a displayName property if it is the same as the job is there?